server/app/rest_api/rest.py

```diff
@@ -88,7 +88,7 @@ class InitializeAPI(Resource):
         }
     })
     def get(self):
-        from app import data, REACTIVE_LIMIT
+        from ..app import data, REACTIVE_LIMIT
```

Comment:
for what it's worth, we had a lengthy argument about this in hca-land, and we eventually settled on no relative imports UP.  only relative imports down or sideways.  i think there's some merit in that decision.

Reply:
I'd be interested in seeing the discussion, do you have a link? I'm happy to make the change, just want to be able to articulate the reasons later.